device/globals/background.gd

```diff
@@ -5,7 +5,7 @@ export var action = "walk"
 func input(event):
 	if event is InputEventMouseButton && event.pressed:
 		if (event.button_index == 1):
-			get_tree().call_group_flags(SceneTree.GROUP_CALL_DEFAULT, "game", "clicked", self, get_position() + Vector2(event.position.x, event.position.y))
+			get_tree().call_group_flags(SceneTree.GROUP_CALL_DEFAULT, "game", "clicked", self, get_position() + Vector2(event.position.x, event.position.y), event)
```

Comment:
`Vector2(event.position.x, event.position.y)` is better written as `event.position`

Reply:
You should also pass the event object from the equivalent function call in background_area.gd:

https://github.com/godotengine/escoria/blob/master/device/globals/background_area.gd#L10

Reply:
In my defense I offer the court that the original `Vector2` code wasn't mine, though I plead guilty of not having fixed it ;)

Of course it should be clearer.

Reply:
I believe you. :slightly_smiling_face: This script has remained unchanged since Escoria was open sourced in August 2016, so my guess is, at some point the x and y coordinates were separate properties of the event object, making it necessary to wrap them in a `Vector2D` type.